src/vue/backend/src/templates/GuidedSetupContentCard.vue

```diff
@@ -60,8 +60,8 @@ export default {
 <style lang="scss">
     @import '~@idxbrokerllc/idxstrap/dist/styles/components/dialog';
 
```

Comment:
Buttons are missing prop `theme`, causing console errors and design issues 
`Invalid prop: custom validator check failed for prop "theme".`
 
![Screen Shot 2021-02-25 at 8 43 34 AM](https://user-images.githubusercontent.com/25048313/109186525-dd0d0980-7745-11eb-9e43-57863d43c4e5.png)

Reply:
That's super odd - the theme has a default of "primary"

Reply:
I agree, it seemed weird to me as well. Could be a VCL build error?